packages/cron/test/pins-backup.spec.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-env mocha */
+import { createPsaPinRequest, createUser, createUpload, createUserAuthKey, randomCid } from '@web3-storage/db/test-utils'
+import Backup from '../src/jobs/pins-backup.js'
+import assert from 'assert'
+import { getCluster, getDBClient, getPg } from '../src/lib/utils.js'
+import sinon from 'sinon'
+import fetch from '@web-std/fetch'
+
+global.fetch = fetch
+
+const env = {
+  DEBUG: 'backup:pins',
+  ENV: 'dev',
+  CLUSTER_API_URL: 'http://localhost:9094',
+  CLUSTER_IPFS_PROXY_API_URL: 'http://127.0.0.1:9095/api/v0/',
```

Comment:
I think you probably want to connect directly to one of the nodes where the content is pinned. If you use the HTTP API via the DNS address you'll end up connecting to a random node and asking it to fetch the DAG (if it doesn't have it). There's no GC on these nodes so we want to avoid copying other nodes in the cluster where the data has not been allocated.

I'm also not certain the `api/v0/dag/export` route has been opened up.

In https://github.com/nftstorage/backup we use libp2p to connect to all nodes in the cluster and transfer via bitswap - only the nodes that are hosting the content will respond.

You could maybe do this with dagula nowadays if you look up which peer the content is hosted on.